EIPS/eip-2569.md

```diff
@@ -350,7 +350,7 @@ _tokenImageSvg = svgCode
 
 ## Articles & Discussions
 
-[Image Storage Solution Based on Ethereum Sharding](https://www.deme.app/full#12)
+[Image Storage Solution Based on Ethereum Sharding,2019](https://naturaldao.io/en/collaboration/blog/65-storage-shading.html)
```

Comment:
Submit these as a separate PR and this will auto-merge.

Reply:
> Submit these as a separate PR and this will auto-merge.

I'm sorry but I really don't know how to make them separate.
Does it mean I have to fork ethereum:master again, create a new branch and submit another EIP (let's say EIP-xxxx) for the ETH staking proposal, and then delete the file eip-2794.md in the old repository?

Reply:
It depends on how you are going about this.  What you *want* is one fork of the `ethereum` repository with two branches on it, both of which build off of `ethereum/master`.  One of the branches would contain one of the change sets and the other would contain the other change set.  How you get there from here depends on what tool you are using.

Once you have the two change sets on different branches, you would then create one or more GitHub Pull Requests (or update your existing ones) so that each is a pull request of a different branch.

Reply:
I always do this on my PC's browser. So now I'd like to keep EIP-2569 with no change. How can I move EIP-2794 into a new branch and keep the same number?
Thanks indeed!

Reply:
The *easiest* way to resolve the issue using the GitHub UI would be to navigate to https://github.com/ethereum/EIPs/tree/master/EIPS and click at the top to create a new file.  Name it `eip-2794.md` and copy the latest version of it into that file.  Commit and create pull request on the following screen.

Then go to https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2569.md and click the edit icon on top of the file and make remake the changes you made to EIP-2569.  At the bottom of the page when you commit, **make sure to tell it to create a new branch!**.  Proceed to create a pull request on the following screen.

Reply:
> Then go to https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2569.md and click the edit icon on top of the file and make remake the changes you made to EIP-2569. **At the bottom of the page when you commit, **make sure to tell it to create a new branch!**. Proceed to create a pull request on the following screen.**

There's no such a selection and it automatically created a new branch. After it's done, I created a pull request.
Now I'm here: https://github.com/ethereum/EIPs/pull/2815. And it says:
**You’re all set—the zhous:patch-3 branch can be safely deleted.**
Can I really delete the branch now?

Reply:
Yes.

Reply:
However, you still have two files in this EIP, so it would appear that you didn't include all of your changes in the previous one.  Just repeat that same process for the changes to eip-2569.md that are present here.

Reply:
Dear Sir, I have made them one in branch "https://github.com/zhous/EIPs/tree/patch-5/EIPS"(eip-2569.md), and another one in branch "https://github.com/zhous/EIPs/tree/patch-4/EIPS"(eip-2794.md). Now I'm totally lost...